styles.css

```
@@ -0,0 +1,183 @@
+a {
+  text-decoration: none;
+}
+
+.header {
+  width: 375px;
+  height: 92px;
+  background-color: #303030;
+  display: flex;
+  justify-content: space-between;
+}
+
+.name {
+  color: white;
+  height: 24px;
+  font-family: Roboto, serif;
+  font-size: 18px;
+  font-style: normal;
+  font-weight: 700;
+  line-height: 24px;
+  letter-spacing: 0;
+  text-align: center;
+  position: relative;
+  top: 20px;
+  left: 10px;
+}
+
+.name p {
+  height: 24px;
+  width: 187px;
+}
+
+.burger {
+  height: 24px;
+  width: 20px;
+  margin-left: 40px;
+  margin-right: 5px;
+  margin-bottom: 15px;
+  border-radius: 0;
+  align-self: center;
+  position: relative;
+  top: 10px;
+  right: 10px;
+}
+
+.burger a {
+  background-color: white;
+  margin-bottom: 5px;
+  padding-bottom: 0;
+}
+
+.main {
+  border-top: 1px solid slateblue;
+  color: white;
+  background-color: #151515;
```

- [ ] The background color you have for this section of the HTML is not the same as that of your chosen template in Figma. Consider using the correct background color for this section. Below is the correct background color.

```suggestion
  background-color: #1C1A19;
```

---

Done.